resources/sdk/purecloudjava/templates/Configuration.mustache

```diff
@@ -20,5 +36,245 @@ public class Configuration {
+    if (logRequestBody != null && !logRequestBody.isEmpty()) {
+        logger.setLogRequestBody(parser.getBool("logging", "log_request_body"));
+    }
+    String logResponseBody = parser.getString("logging", "log_response_body");
+    if (logResponseBody != null && !logResponseBody.isEmpty()) {
+        logger.setLogResponseBody(parser.getBool("logging", "log_response_body"));
+    }
+    // General
+    String host = parser.getString("general", "host");
+    if (host != null && !host.isEmpty()) {
+        defaultApiClient.setBasePath(host);
+    }
+    String liveReloadConfig = parser.getString("general", "live_reload_config");
+    if (liveReloadConfig != null && !liveReloadConfig.isEmpty()) {
+        defaultApiClient.setAutoReloadConfig(parser.getBool("general", "live_reload_config"));
+    }
+    // Re-authentication
+    String refreshAccessToken = parser.getString("reauthentication", "refresh_access_token");
+    if (refreshAccessToken != null && !refreshAccessToken.isEmpty()) {
+        defaultApiClient.setShouldRefreshAccessToken(parser.getBool("reauthentication", "refresh_access_token"));
+    }
+    String refreshTokenWaitTime = parser.getString("reauthentication", "refresh_token_wait_max");
+    if (refreshTokenWaitTime != null && !refreshTokenWaitTime.isEmpty()) {
+        defaultApiClient.setRefreshTokenWaitTime(parser.getInt("reauthentication", "refresh_token_wait_max"));
+    }
+    // Retry
+    String maxRetryTimeSec = parser.getString("retry", "retry_wait_max");
+    if (maxRetryTimeSec != null && !maxRetryTimeSec.isEmpty()) {
+        ApiClient.RetryConfiguration retryConfiguration = defaultApiClient.getRetryConfiguration();
+        retryConfiguration.setMaxRetryTimeSec(parser.getInt("retry", "retry_wait_max"));
+        defaultApiClient.setRetryConfiguration(retryConfiguration);
+    }
+    String retryMax = parser.getString("retry", "retry_max");
+    if (retryMax != null && !retryMax.isEmpty()) {
+      ApiClient.RetryConfiguration retryConfiguration = defaultApiClient.getRetryConfiguration();
+      retryConfiguration.setRetryMax(parser.getInt("retry", "retry_max"));
+      defaultApiClient.setRetryConfiguration(retryConfiguration);
+    }
+  }
+
+  public String getConfigHash() {
+    try (FileInputStream fis = new FileInputStream(new File(configFilePath))) {
+      MessageDigest md = MessageDigest.getInstance("MD5");
+      byte[] bytesBuffer = new byte[1024];
+      int bytesRead = -1;
+      while ((bytesRead = fis.read(bytesBuffer)) != -1) {
+        md.update(bytesBuffer, 0, bytesRead);
+      }
+      byte[] hashedBytes = md.digest();
+      return new String(hashedBytes, StandardCharsets.UTF_8);
+    } catch (Exception e) {
+      return "";
+    }
+  }
+
+  public void runConfigChecker() {
+    while (true) {
+        try {
+          Thread.sleep(30 * 1000);
+        } catch (InterruptedException e) {
+          Thread.currentThread().interrupt();
+          break;
+        }
+        if (!autoReloadConfig) {
```

Comment:
Seems like this is never being set to false using a config file. I tried having it set to false before starting the application and while it was running but it stayed true. That was with INI and JSON

Reply:
This should be fixed

Reply:
Still have the issue with this. It's always true no matter what I do with the config

Reply:
I realized that autoReloadConfig was only being handled initially.   I added a couple of helper methods and this should be fixed now.